test/test_link.rb

```diff
@@ -40,6 +40,27 @@ module SDF
                 assert Eigen::Quaternion.Identity.approx?(i.pose.rotation)
                 assert Eigen::MatrixX.from_a([1,0,0, 0,1,0, 0,0,1], 3, 3, false), i.inertia
             end
+
+            it "returns a true kinematic link" do
+                xml = REXML::Document.new("<link><pose>1 2 3 0 0 2</pose><kinematic>true</kinematic></link>").root
+                link = Link.new(xml)
+                k = link.kinematic
+                assert k
+            end
+
+            it "returns a false kinematic link" do
+                xml = REXML::Document.new("<link><pose>1 2 3 0 0 2</pose><kinematic>false</kinematic></link>").root
+                link = Link.new(xml)
+                k = link.kinematic
+                assert !k
```

Comment:
Just condense the two lines as `assert !link.kinematic?`.

You could even condense the three lines. But that's your call.